acme/build.gradle

```diff
@@ -0,0 +1,31 @@
+dependencies {
+    annotationProcessor("io.micronaut:micronaut-inject-java:${micronautVersion}")
+    annotationProcessor "io.micronaut.docs:micronaut-docs-asciidoc-config-props:$micronautDocsVersion"
+    implementation("io.micronaut:micronaut-http:${micronautVersion}")
+    implementation("io.micronaut:micronaut-http-server:${micronautVersion}")
+    implementation("io.micronaut:micronaut-http-server-netty:${micronautVersion}")
+    implementation("io.micronaut:micronaut-inject:${micronautVersion}")
+    compileOnly("io.micronaut:micronaut-inject-java:${micronautVersion}")
```

Comment:
Should be able to pull the bom in and drop a bunch of `micronautVersion`s

Reply:
Done and done 511bda5....good call. 👍